lib/crowbar/client/command/node/list.rb

```diff
@@ -73,6 +73,10 @@ def content_from(request)
                 )
               end
             end
+
+            result.sort do |x, y|
+              x.values.join <=> y.values.join
```

Comment:
Ah OK, [the order is deterministic since 1.9.2](http://stackoverflow.com/questions/5208512/ruby-what-is-the-order-of-keys-values-returned-by-hash-keys-and-hash-values-met).  I missed that change.  So it will sort first by name then by alias.  I think that's fine, but it would be worth adding a short comment explaining that, since it's a non-obvious subtlety.